src/ConstructStyleSheet.js

```diff
@@ -83,16 +60,23 @@ export default class ConstructStyleSheet {
       deferredStyleSheets.push(basicStyleElement);
     }
 
-    const nativeStyleSheet = basicStyleElement.sheet;
-
     // A support object to preserve all the polyfill data
-    sheetMetadataRegistry.set(nativeStyleSheet, {
+    sheetMetadataRegistry.set(this, {
       adopters: new Map(),
       actions: [],
       basicStyleElement,
     });
+  }
+
+  get cssRules() {
+    if (!sheetMetadataRegistry.has(this)) {
+      throw new Error(
+        "Cannot read 'cssRules' on non-constructed CSSStyleSheets.",
```

Comment:
Where does this behavior come from? In Chrome you can read `cssRules`, try `document.styleSheets[0].cssRules`.

Reply:
The same check is in all of the methods, so I added it here to. Basically, we throw if anybody tries to call a method or read a property directly from the prototype without first calling `new CSSStyleSheet()`. Chrome throws an `Illegal invocation` error - I can update the messaging it to match.
```
CSSStyleSheet.prototype.cssRules;
CSSStyleSheet.prototype.replace();
```

Reply:
Please do update the messaging. We want to be as close as possible to what Chrome does for the time being.